myapp/app/controllers/application_controller.rb

```diff
@@ -46,6 +50,10 @@ def _render_500(exception = nil)
   end
 
   def current_user
-    @current_user ||= session[:user]
+    if session[:user]
```

Comment:
Just for sharing.
You can also write it using the ternary operator for this case.

``` rb
session[:user] ? (@current_user ||= User.find(session[:user]['id'])) : nil
```

Reply:
oh nice!
thanks